yt_dlp/extractor/israelnationalnews.py

```diff
@@ -0,0 +1,47 @@
+from .common import InfoExtractor
+from ..utils import ExtractorError, traverse_obj
+
+
+class IsraelNationalNewsIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?israelnationalnews\.com/news/(?P<id>[0-9]+)'
+    _TESTS = [{
```

Comment:
Since u changed it to playlist, test needs to be updated accordingly

Reply:
I think the test still works in this case (I happen to do the same thing in my PR for when it only has one video https://github.com/yt-dlp/yt-dlp/pull/5088).

Unless you think the test should be a playlist test regardless.

Reply:
It should be a playlist test anyway. If it works like this, that is a bug in the test runner imo

Reply:
Alright, updated as of https://github.com/yt-dlp/yt-dlp/pull/5089/commits/3eda01301ff5a8f36fdae81d634bfd8daa9556c8